P6-Setup-App-Structure/content.md

```diff
@@ -1,63 +1,68 @@
+It is common to see multiple types of container view controllers in a single app - we will explore this idea when adding the image filter feature to **Makestagram** later on.
 
-#Setting the UITabBarController up
+#Setting Up the UITabBarController
 
-As mentioned earlier, we will skip the login screen for now. That means that the `UITabBarController` is the first View Controller we need to add to our app. We also want the Tab Bar Controller to be the initial View Controller of our app.
+Since we won't be adding the login screen until later, the first screen we need to setup is our Tab Bar Screen.
 
-The Xcode project template has a simple `UIViewController` as its initial View Controller.
-Let's first go ahead and delete it.
+The **Makestagram** project template that you downloaded earlier currently has one screen, a simple `UIViewController`. We don't need that screen, so let's go ahead and delete it.
 
 > [action]
-Open *Main.storyboard*. Select the existing View Controller; after you have selected it you should see a blue border surrounding it:
+Open *Main.storyboard*. Select the existing view controller; after you have selected it you should see a blue border surrounding it:
 ![image](selected_vc.png)
 >
-Hit the *delete* key to remove this View Controller. Now you should see a blank Storyboard:
+Hit the *delete* key to remove this view controller. Now you should see a blank storyboard:
 ![image](blank_storyboard.png)
 
+Let's also delete the screen's accompanying source file called ViewController.swift.
+
+
+> [action]
+Delete the *ViewController.swift* file:
+![image](delete_vc.png)
+![image](delete_vc_trash.png)
+
 Now we can add the `UITabBarController`.
 
 > [action]
-Drag the `UITabBarController` from the component library into the Storyboard:
+Drag the `UITabBarController` from the component library into the storyboard:
 ![image](add_tab_bar.png)
 
-Now we have set up the Container View Controller for our app - time to run it?
+We have now set up the container view controller for our app - time to run it?
 
-If you run the app now, you will see a blank, black screen, after the splash image disappears. You will also see the following error in the console:
+Nope! If you run the app now, after the splash image disappears, you will see a blank black screen. You will also see the following error in the console:
 
 > Makestagram[76593:9371786] Failed to instantiate the default view controller for UIMainStoryboardFile 'Main' - perhaps the designated entry point is not set?
 
-**Why are you seeing this error?**
+**Why Are We Seeing This Error?**
+
+When working with storyboards, we have to designate an entry point, or more simply, we have to tell the compiler what view controller should appear first (we call this view controller the *Initial View Controller*). When the compiler doesn't know which view controller to show first (a.k.a. when the entry point is not set), it defaults to showing an empty black screen.
 
```

Comment:
When I was first getting started, I remember this specific issue giving me headaches on multiple occasions. I figured diving a little bit deeper into why it occurred might prevent some students from getting tripped up by it -- still want me to get rid of it?